closure/templates/test/BUILD

```diff
@@ -18,17 +18,22 @@ licenses(["notice"])  # Apache 2.0
 
 load("//closure:defs.bzl", "closure_js_binary")
```

Comment:
Please copy the existing tests to `closure/templates/test/legacy` so we don't lose coverage here. I'm not sure if we can remove `closure_js_library` anytime soon...